docs/quickstart.md

```diff
@@ -0,0 +1,164 @@
+<!--[metadata]>
++++
+title = "Quickstart Guide"
+description = "Getting started with Docker Compose"
+keywords = ["documentation, docs,  docker, compose, orchestration, containers"]
+[menu.main]
+parent="smn_workw_compose"
+weight=3
++++
+<![end-metadata]-->
+
+
+## Quick start
+
+Let's get started with a walkthrough of getting a simple Python web app running
+on Compose. It assumes a little knowledge of Python, but the concepts
+demonstrated here should be understandable even if you're not familiar with
+Python.
+
+### Installation and set-up
+
+First, [install Docker and Compose](install.md).
+
+Next, you'll want to make a directory for the project:
+
+    $ mkdir composetest
+    $ cd composetest
+
+Inside this directory, create `app.py`, a simple Python web app that uses the Flask
+framework and increments a value in Redis. Don't worry if you don't have Redis installed, docker is going to take care of that for you when we [define services](#define-services):
+
+    from flask import Flask
+    from redis import Redis
+
+    app = Flask(__name__)
+    redis = Redis(host='redis', port=6379)
+
+    @app.route('/')
+    def hello():
+        redis.incr('hits')
+        return 'Hello World! I have been seen %s times.' % redis.get('hits')
+
+    if __name__ == "__main__":
+        app.run(host="0.0.0.0", debug=True)
+
+Next, define the Python dependencies in a file called `requirements.txt`:
+
+    flask
+    redis
+
+### Create a Docker image
+
+Now, create a Docker image containing all of your app's dependencies. You
+specify how to build the image using a file called
+[`Dockerfile`](http://docs.docker.com/reference/builder/):
+
+    FROM python:2.7
+    ADD . /code
+    WORKDIR /code
+    RUN pip install -r requirements.txt
+    CMD python app.py
+
+This tells Docker to:
+
+* Build an image starting with the Python 2.7 image.
+* Add the current directory `.` into the path `/code` in the image.
+* Set the working directory to `/code`.
+* Install the Python dependencies.
+* Set the default command for the container to `python app.py`
+
+For more information on how to write Dockerfiles, see the [Docker user guide](https://docs.docker.com/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the [Dockerfile reference](http://docs.docker.com/reference/builder/).
+
+You can build the image by running `docker build -t web .`.
+
+### Define services
+
+Next, define a set of services using `docker-compose.yml`:
+
+    web:
+      build: .
+      ports:
+       - "5000:5000"
+      volumes:
+       - .:/code
+      links:
+       - redis
+    redis:
+      image: redis
+
+This template defines two services, `web` and `redis`. The `web` service:
```

Comment:
Thinking out loud; I've seen people getting confused over the naming of the services. They tend to think that a service must be named after the image that's used, or that the name has special meaning. Perhaps mention that services can have any name you like